superset/views/utils.py

```diff
@@ -101,15 +101,18 @@ def get_viz(
 
 def get_form_data(slice_id=None, use_slice_data=False):
```

Comment:
While we're at it, let's put in type annotations here; I was surprised to see that this was returning a tuple.